src/FirePokemon.java

```diff
@@ -0,0 +1,4 @@
+public class FirePokemon {
+	public FirePokemon(String charrizard, int i, int i1, String firenougats, String grrrrrrrrrrrrrrr) {
```

Comment:
Grappig om te zien welke namen IntelliJ automatisch aan je parameters geeft als je de constructor laat genereren. Maar dit zijn natuurlijk geen bruikbare namen, dus als je hiermee aan de slag gaat, kies dan goede namen om deze te vervangen.

Zie ook dat het soms handig kan zijn om iets door IntelliJ te laten genereren, zoals bij imports, maar het is vaak niet de oplossing voor een probleem, zoals deze lege constructor. Dit lost de compiler fout op, maar als je de code runt, crasht het alsnog (je krijgt er een runtime error voor terug).